docs/user/security/securing-communications/index.asciidoc

```diff
@@ -4,127 +4,189 @@
+
+TLS requires X.509 certificates to authenticate the communicating parties and perform encryption of data-in-transit. Each certificate
+contains a public key and has an associated -- but separate -- private key; these keys are used for cryptographic operations. {kib}
+supports certificates and private keys in PEM or PKCS#12 format.
 
 [[configuring-tls-browser-kib]]
 ==== Encrypting traffic between the browser and {kib}
 
-NOTE: You do not need to enable {security-features} for this type of encryption. 
+NOTE: You do not need to enable the {es} {security-features} for this type of encryption.
 
 . Obtain a server certificate and private key for {kib}.
 +
 --
-{kib} supports certificates/keys in both PKCS #12 key stores and PEM format.
+{kib} will need to use this "server certificate" and corresponding private key when receiving connections from web browsers.
 
-When you obtain a certificate, you must do at least one of the following:
+When you obtain a server certificate, you must set its subject alternative name (SAN) correctly to ensure that modern web browsers with
+hostname verification will trust it. You can set one or more SANs to the {kib} server's fully-qualified domain name (FQDN), hostname, or IP
+address. When choosing the SAN, you should pick whichever attribute you will be using to connect to {kib} in your browser, which is likely
+the FQDN.
 
-.. Set the certificate's `subjectAltName` to the hostname, fully-qualified domain name (FQDN), or IP address of the {kib} server.
 
-.. Set the certificate's Common Name (CN) to the {kib} server's hostname or FQDN. Using the server's IP address as the CN does not work.
-
-You may choose to generate a certificate and private key using {ref}/certutil.html[the {es} certutil tool]. If you already used certutil to
-generate a certificate authority (CA), you would generate a certificate/key for Kibana like so (using the `--dns` param to set the
-`subjectAltName`):
+You may choose to generate a certificate signing request (CSR) and private key using the {ref}/certutil.html[`elasticsearch-certutil`] tool.
+For example:
 
 [source,sh]
 --------------------------------------------------------------------------------
-bin/elasticsearch-certutil cert --ca elastic-stack-ca.p12 --name kibana --dns localhost
+bin/elasticsearch-certutil csr -name kibana-server -dns some-website.com,www.some-website.com
 --------------------------------------------------------------------------------
 
-This will generate a certificate and private key in a PKCS #12 keystore named `kibana.p12`.
+This will produce a ZIP archive named `kibana-server.zip`. Extract that archive to obtain the PEM-formatted CSR (`kibana-server.csr`) and
+unencrypted private key (`kibana-server.key`). In this example, the CSR has a common name (CN) of `"kibana-server"`, a SAN of
+`"some-website.com"`, and another SAN of `"www.some-website.com"`.
```

Comment:
Tiny nit: I think the following change would be easier to ready, as the content of the code-blocks are not shown in quotes in the example above:
```suggestion
`some-website.com`, and another SAN of `www.some-website.com`.
```